packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -429,6 +429,26 @@ export enum AccountRecovery {
   NONE,
 }
 
+/**
+ * Device tracking settings
+ * @see https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-device-tracking.html
+ */
+export interface DeviceConfiguration {
```

Comment:
We're using `Settings` as the suffix, like `EmailSettings`. Can we use `DeviceSettings` instead?

Would `DeviceTracking` be better? (Is `challengeRequiredOnNewDevice` part of device tracking?)

Reply:
I have renamed DeviceConfiguration to DeviceTracking.
Regarding challengeRequiredOnNewDevice that is a part of the deviceConfiguration spec: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cognito-userpool-deviceconfiguration.html#cfn-cognito-userpool-deviceconfiguration-challengerequiredonnewdevice